main/src/ca/uwaterloo/flix/language/ast/SourcePosition.scala

```diff
@@ -1,25 +1,21 @@
 package ca.uwaterloo.flix.language.ast
 
 import ca.uwaterloo.flix.language.ast.Ast.Source
-import org.parboiled2.ParserInput
-
-import javax.annotation.Nullable
 
 object SourcePosition {
   /**
     * Represents an unknown source position.
     */
-  val Unknown: SourcePosition = SourcePosition(Source(Ast.Input.Text("<unknown>", "", stable = true), Array.emptyCharArray, stable = true), 0, 0, null)
+  val Unknown: SourcePosition = SourcePosition(Source(Ast.Input.Text("<unknown>", "", stable = true), Array.emptyCharArray, stable = true), 0, 0)
```

Comment:
I smell danger. If the string is "" and you start indexing into it, what happens?

Reply:
That gets handled in `getLine` 👍